docs/development/database_schema.md

```diff
@@ -155,43 +155,11 @@ def run_upgrade(
 Boolean columns require special treatment, since SQLite treats booleans the
 same as integers.
 
-There are three separate aspects to this:
-
- * Any new boolean column must be added to the `BOOLEAN_COLUMNS` list in
+Any new boolean column must be added to the `BOOLEAN_COLUMNS` list in
    `synapse/_scripts/synapse_port_db.py`. This tells the port script to cast
    the integer value from SQLite to a boolean before writing the value to the
    postgres database.
 
- * Before SQLite 3.23, `TRUE` and `FALSE` were not recognised as constants by
-   SQLite, and the `IS [NOT] TRUE`/`IS [NOT] FALSE` operators were not
-   supported. This makes it necessary to avoid using `TRUE` and `FALSE`
-   constants in SQL commands.
-
-   For example, to insert a `TRUE` value into the database, write:
-
-   ```python
-   txn.execute("INSERT INTO tbl(col) VALUES (?)", (True, ))
-   ```
```

Comment:
Should we file an issue to track doing this for any current queries? I don't know how easy that would be to find, but I definitely know I've written a few where I purposefully pass in `True` or `False` as parameters...

Reply:
Yes, that's what I mean.

I feel less strongly about converting `=` to `IS` than just inlining the variables: I think reducing the number of placeholders makes the queries clearer.

Reply:
Right on I'll open an issue for that.